devel/hello-world.lua

```diff
@@ -1,31 +1,61 @@
--- Test lua viewscreens.
---[====[
+-- A basic example to start your own gui script from.
+--@ module = true
 
-devel/hello-world
-=================
-A basic example for testing, or to start your own script from.
+local gui = require('gui')
+local widgets = require('gui.widgets')
 
-]====]
-local gui = require 'gui'
+local HOVER_FRAME = copyall(gui.BOUNDARY_FRAME)
+HOVER_FRAME.signature_pen = false
```

Comment:
If this is hiding the "DFHack" text, I don't think we should encourage this by default

Reply:
haha, that's a good point